In [45]:
import numpy as np

Parameters for argon gas under 300 Kelvin

In [46]:
NA = 6.02214e23  #/mol
T = 300  #K
rho = 1.784  #kg/m^3
N = 200
M = 0.039948  #kg/mol

In [47]:
#Calculate the cell length
m = (N / NA) * M  #kg
V = (m / rho) * (1e27)  #nm^3
L = V**(1.0 / 3.0)  #nm
print(L)  #nm
print(L / (200**0.33))  #Average distance on one axis of each atom

19.519099582236738
3.3971903454338084


Increase the pressure, cell length set to be 6 nm to test Langevin thermostat. 

In [48]:
kB = 1.380649e-23  #J/K
ep = kB * 120.0  #J
print(ep)

1.6567788e-21


The Lennard-Jones parameters are: $\sigma = 0.34\ nm$, $\frac{\epsilon}{k_B}= 120\ K$, $\epsilon = 1.66 \times 10^{-21}\ J$